File: benchmark/Python 2.7/循环分支累加耗时.py
import time

start = time.time()

sum = 0
for i in xrange(10000000):
	if i <= 2500000:
		sum += i
	elif i <= 5000000:
		sum += i+1
	elif i <= 7500000:
		sum += i+2
	else:
		sum += i+3
print(sum, "\n")

print(time.time()-start)

File: benchmark/Lua 5.3/循环分支累加耗时.lua
local start = os.clock()

local sum = 0
for i = 0, 10000000-1, 1 do
	if i <= 2500000 then
		sum = sum+i
	elseif i <= 5000000 then
		sum = sum+i+1
	elseif i <= 7500000 then
		sum = sum+i+2
	else
		sum = sum+i+3
	end
end
print(sum, "\n");

print(os.clock()-start, "\n")
